service/recognition/noise_reduction_model.py

```diff
@@ -0,0 +1,35 @@
+"""
+Noise-reduction module
+"""
+
+from typing import Any
+
+import torch
+from df.enhance import enhance, init_df
+
+from .model import RecognitionModel
+
+
+class NoiseReductionModel(RecognitionModel):
```

Comment:
The best option in my opinion would be to create a separate abstract class for noise reduction models. The expected result for us in this case will be the composition of the results

```python
expected result = recognize(denoise(input_data))
```

Reply:
it is a good idea, but the Nemo model (for transcribing audio) can't work directly with file object or bytes. We need to give a file path.